src/parser.rs

```diff
@@ -159,4 +160,407 @@ mod tests {
         let keys = parse_binding(binding);
         assert_eq!(keys, vec![Keys::LeftCtrl as u32, Keys::LeftAlt as u32, Keys::Delete as u32]);
     }
+    #[test]
```

Comment:
I think we may be able to get away with having a single test that utilises a loop to go through a collection of pairs of expected/actual.

Reply:
Reworked the tests, take a look at the new code please